### PR TITLE
Update obsolete address and remove obsolete translations

### PR DESCRIPTION
**Pull request by flodolo**
Fixes issue #1663 

This PR does a few things, some of which might not be OK (I kept separate commits, so I can undo single parts).
* It removes 3 obsolete files, per [this comment](https://github.com/mozilla/legal-docs/issues/1663#issuecomment-982924543): `better_web_terms.md`, `mobile_partner_website_branding_terms.md`, `mobile_partner_website_prototype_agreement.md`.
* It removes obsole translations of Firefox Cloud Services, either from 2014 or 2017. I think these were mostly for Firefox OS, the English document was last updated in 2019. Affected locales: af, ar, cs, el, hi, hr, hu, mk, ro, sr, sw, ta, tl-PH, xh, zu.
* It updates the address in websites_out.md and Firefox Cloud Services TOS for the locales that remain.

### CODE REVIEW

**Comment by flodolo** [comment]
Sorry, forgot one more: it also removes `en-DE/klar_privacy_notice.md`, which would be #1656

**Review by mafeld — APPROVED** [review]

**Comment by flodolo** [comment]
CC @pmac as this removes a bunch of files. Probably good to let it live on DEV for a bit, to see if anything explodes.

File: es-ES/websites_tou.md [hunk 1]
```diff
@@ -90,22 +90,21 @@ LOS MENSAJES SE FACILITAN "TAL Y COMO ESTÁN", CON TODOS SUS FALLOS. EN LA MEDID
...
-Fax: 650-903-0875  
-Avisos legales en mozilla.com
-
+Attn: Mozilla – Legal Notices
+2 Harrison St # 175,
+San Francisco, CA 94105
+USA
+Teléfono: 650-903-0800
+Fax: 650-903-0875
+Legal-notices at mozilla.com
```
Comment (pdehaan) [inline es-ES/websites_tou.md:110]: Per https://daringfireball.net/projects/markdown/syntax
> _"When you do want to insert a `<br />` break tag using Markdown, you **end a line with two or more spaces**, then type return."_

I wonder if this will cause the address to now run on one line.
As close as I can tell, these "[locale]/websites_tou.md" files are used in https://www.mozilla.org/en-US/about/legal/terms/mozilla/, so might be worth confirming the layouts after the pages get redeployed to make sure the addresses are still properly formatted.

Comment (flodolo) [reply es-ES/websites_tou.md:110]: Shut, it does :-( 

https://www-dev.allizom.org/es-ES/about/legal/terms/mozilla/

I used a different editor for updating the other docs, I really think relying on double spaces for new lines is a terrible idea, because at some point someone will remove them (like I did).

Comment (flodolo) [reply es-ES/websites_tou.md:110]: This should be fixed by #1670, waiting for automation to pick up the changes

Comment (pdehaan) [reply es-ES/websites_tou.md:110]: > _"I really think relying on double spaces for new lines is a terrible idea"_

💯 , but sadly it's now forever in the Markdown spec.
Although double checking against the Commonmark spec, there might be an alternate syntax option of EOL backslash: https://commonmark.org/help/tutorial/03-paragraphs.html
> _"For a line break, add either a backslash `\`, or two blank spaces at the end of the line."_

But then double-double checking that against the [Markdown dingus](https://daringfireball.net/projects/markdown/dingus), dashes my hopes and dreams yet again.

```md
the quick\
brown fox  
lazy dog
```

### HTML (Markdown)
```html
<p>the quick\
brown fox <br />
lazy dog</p>
```

### HTML (Commonmark)
```html
<p>the quick<br>
brown fox<br>
lazy dog</p>
```

TLDR: Depending on which Markdown engine mozilla.org uses, we might be able to use commonmark trailing slash syntax instead of trailing double spaces [but probably best not to, for compatibility].

Comment (pmac) [reply es-ES/websites_tou.md:110]: Also note that Markdown is a superset of HTML and you can just use `<br>` as well. We could test out the backslash though.

**Comment by pmac** [comment]
The website still does link to Better Web, but the page is a 404 now due to the file removal https://www.mozilla.org/en-US/about/legal/

**Comment by mafeld** [comment]
Thanks @pmac. Do you have the link to the .md for that doc? I don't see it. 
I can update it once I know which doc it's pulling from.

**Comment by flodolo** [comment]
Filed an issue (#1676), hopefully it's easier to track
